routes/issues.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const router = express.Router();
```

Comment:
Please move this line after all the `require` statements

Reply:
This comes by default after formatting.

Reply:
Sorry, I did not understand what you mean by "by default after formatting"
What formatting are we referring to here?

Reply:
Refer `tags.js` routes for this
https://github.com/Real-Dev-Squad/website-backend/blob/4e449b42e50c62c612dbccb35589bdc55281376a/routes/tags.js#L1-L9

Reply:
> Please move this line after all the `require` statements

I thought you referred to the extra line that comes after formatting.

And I'm not sure what line you're referring to. ` line after all the require statements` this imports the issues controller.

Reply:
I meant this particular line
```js
const router = express.Router(); 
```
Although I think we can ignore it for now as this is a high priority feature